### PR TITLE
RSpec test view login, logout, homepage, user_profile

### PR DESCRIPTION
**Pull request by thiennv57**

### CODE REVIEW

File: spec/views/devise/log_out_spec.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  feature "Log out" do
+    let!(:user) {FactoryGirl.create :user}
+
+    scenario "Log out" do
+      visit "/users/sign_in"
+      fill_in "Email", with: user.email
+      fill_in "Password", with: user.password
+      click_button "Login"
+      
+      click_link "Profile"
```
Comment (ttuan) [inline spec/views/devise/log_out_spec.rb]: remove this line

File: spec/views/static_pages/home.html_spec.rb [hunk 2]
```diff
@@ -0,0 +1,43 @@
+RSpec.describe User, "Show home page" do
+  feature "Show home page" do
+    let!(:user1){FactoryGirl.create :user}
```
Comment (ttuan) [inline spec/views/static_pages/home.html_spec.rb]: add a space

File: spec/views/static_pages/home.html_spec.rb [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    let!(:user2) {FactoryGirl.create :user}
+
+    before do
+      visit "users/sign_in"      
+      fill_in "Email", with: user1.email
+      fill_in "Password", with: user1.password
+      click_button "Login"
+    end
+
+    scenario "User can see home page when loggein" do
```
Comment (ttuan) [inline spec/views/static_pages/home.html_spec.rb]: logged in

**Comment by binhthanhnguyen** [comment]
okay

**Pull request merged**